maintainers/maintainer-list.nix

```diff
@@ -19199,6 +19199,16 @@
     githubId = 17836748;
     name = "Mason Mackaman";
   };
+  ursuslotor = {
```

Comment:
It's recommended that this matches your GitHub username, BTW.
https://github.com/NixOS/nixpkgs/blob/cf53751a16df6ae52eb3be7019aa9c34017e490b/maintainers/maintainer-list.nix#L34

Reply:
If it's a must, I could change it, but it it's alright, I would like to keep it shorter using `ursuslotor`. This handle is also used in my [email](mailto:me@ursuslotor.xyz), domain, and [other sites](https://gravatar.com/ursuslotor), like [keybase](https://keybase.io/ursuslotor).

P.S. I could put an `aka` in my Github bio if that would suffice.

Reply:
Oh, just saw it. There is already an `@ursuslotor` on [Github](https://github.com/ursuslotor), so indeed, it's better if I change the handle.